src/style.css

```
@@ -0,0 +1,76 @@
+@font-face {
+  font-family: "Roboto";
+  src: url("./fonts/Roboto-Medium.ttf");
+  font-weight: 500;
+}
+
+html {
+  --blue-color: #00acdc;
+}
+
+body {
+  margin: 0;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+header {
+  font-family: "Roboto", sans-serif;
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+}
+
+nav {
+  display: flex;
+  height: 60px;
+}
+
+.link {
+  display: flex;
+  position: relative;
+  align-items: center;
+  
+  text-transform: uppercase;
+
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 14px;
+}
+
+.is-active,
+.link:hover {
```

```suggestion
.link:hover {
```
у .is-active есть класс link